tests/testthat/test-sanitize-filename.R

```diff
@@ -21,3 +21,10 @@ test_that("Sanitized file names contain only alphanumeric characters and dots",
 test_that("If the file name is empty after being sanitized, the default is returned", {
   expect_equal(sanitizeFileName("ℍℍℍℍ ", "default"), "default")
 })
+
+test_that("sanitizeFileName is vectorized", {
+  names <- c("foo.txt", "", "bar.txt")
+  defaults <- sapply(0:(length(names)-1), as.character)
```

Comment:
This isn't a big deal here, but two things:
* In most reusable code it's better to use `seq`, `seq_len`, or `seq_along` instead of `1:n`, because it will probably not return what you want when `n` is less than the first number.
* It's generally best to avoid `sapply` because the return type isn't known ahead of time, and can change depending on the results. Better to use `lapply` or `vapply`. And in this particular case, you don't need any of those, since `as.character` is vectorized.

So I think you'd have something like this:

```R
defaults <- as.character(seq_along(names) - 1)
```